gateway-core/components/org.wso2.carbon.gateway.core/src/main/java/org/wso2/carbon/gateway/core/config/dsl/external/wuml/WUMLBaseListenerImpl.java

```diff
@@ -638,27 +636,42 @@ public void exitLogMediatorCall(WUMLParser.LogMediatorCallContext ctx) {
         dropMediatorFilterAware(logMediator);
     }
 
+    /** statements with 'reply'. It maps to a Respond mediator*/
+    @Override public void exitReturnStatement(WUMLParser.ReturnStatementContext ctx) {
+        Mediator respondMediator = MediatorProviderRegistry.getInstance().getMediator("respond");
+        ParameterHolder parameterHolder = new ParameterHolder();
+        if (ctx.Identifier() != null) {
+            String messageId = ctx.Identifier().getText();
+            parameterHolder.addParameter(new Parameter("messageId", messageId));
```

Comment:
Shall we define constants for frequently use strings like "messageId", "key", "type", etc...